data/versions.csv

```diff
@@ -1,59 +1,95 @@
+"aquifer_data","primary","v1.2","['latest', 'supported']","https://download.bgr.de/bgr/grundwasser/IHME1500/v12/shp/IHME1500_v12.zip",""
+"aquifer_data","archive","v1.2","['latest', 'supported']","https://zenodo.org/records/16946750/files/IHME1500_v12.zip",""
+"lau_regions","archive","2019","['latest', 'supported']","https://zenodo.org/records/16947002",""
+"osm_boundaries","primary","unknown","['latest','supported']","","Overpass API is used to fetch the latest available data"
+"osm_boundaries","archive","unknown","['latest','supported']","https://zenodo.org/records/16992755/files/osm_boundaries.zip",""
+"tyndp","primary","2024","['latest','supported']","https://2024-data.entsos-tyndp-scenarios.eu/files/scenarios-inputs",""
+"tyndp","archive","2024","['latest','supported']","https://zenodo.org/records/16759672/files",""
+"powerplants","primary","0.7.1","['latest','supported']","https://raw.githubusercontent.com/PyPSA/powerplantmatching/refs/tags/v0.7.1/powerplants.csv","Part of the `powerplantmatching` package and versioned on GitHub, i.e. no dedicated 'archive' entry."
+"powerplants","primary","0.7.0","['supported']","https://raw.githubusercontent.com/PyPSA/powerplantmatching/refs/tags/v0.7.0/powerplants.csv","Part of the `powerplantmatching` package and versioned on GitHub, i.e. no dedicated 'archive' entry."
+"costs","primary","v0.13.3","['latest','supported']","https://raw.githubusercontent.com/PyPSA/technology-data/v0.13.3/outputs","Part of the `technologydata` repository and versioned on GitHub, i.e. no dedicated 'archive' entry."
+"costs","primary","v0.13.2","['partially-supported']","https://raw.githubusercontent.com/PyPSA/technology-data/v0.13.2/outputs","Part of the `technologydata` repository and versioned on GitHub, i.e. no dedicated 'archive' entry."
+"country_runoff","archive","2025-08-13","['latest','supported']","https://zenodo.org/records/16849356/files/era5-runoff-per-country.csv",""
+"country_runoff","build","unknown","['latest','supported']","","Latest dataset built using ERA5 runoff data. This takes a very long time to build."
+"country_hdd","archive","2025-08-13","['latest','supported']","https://zenodo.org/records/16849356/files/era5-HDD-per-country.csv",""
+"country_hdd","build","unknown","['latest','supported']","","Latest dataset built using ERA5 runoff data. This takes a very long time to build."
+"natura","archive","2025-08-15","['latest','supported']","https://zenodo.org/records/16881818/files/natura.tiff",""
+"natura","archive","v0.8.2","['supported']","https://zenodo.org/records/16874772/files/natura.tiff",""
+"natura","build","unknown","['latest','supported']","https://sdi.eea.europa.eu/datashare/s/tWpGXaWdWGYcqsL/download",""
+"bfs_road_vehicle_stock","archive","2024-03-13","['latest','supported']","https://raw.githubusercontent.com/PyPSA/pypsa-eur/5b5d308bf70f15dd0b107d8a19c121093dcdd5bd/data/gr-e-11.03.02.01.01-cc.csv"

```

Comment:
```suggestion
"bfs_road_vehicle_stock","archive","2024-03-13","['latest','supported']","https://raw.githubusercontent.com/PyPSA/pypsa-eur/5b5d308bf70f15dd0b107d8a19c121093dcdd5bd/data/gr-e-11.03.02.01.01-cc.csv",""
```
Check to make sure that the csv file is well formed, e.g. can be read properly by GitHub. You can check this by opening the file to check if there is an error message, e.g.: https://github.com/open-energy-transition/pypsa-eur/blob/21f22f6c657bb6f4d021b647b52988074c1a1a35/data/versions.csv